Jenkinsfile

```diff
@@ -181,28 +184,84 @@ pipeline {
+          }
           when {
             beforeAgent true
-            tag pattern: '@elastic/apm-rum@\\d+\\.\\d+\\.\\d+$', comparator: 'REGEXP'
+            beforeInput true
+            allOf {
+              branch 'master'
+              expression { return params.release }
+            }
           }
           stages {
-            stage('Opbeans') {
-              environment {
-                REPO_NAME = "${OPBEANS_REPO}"
+            stage('Notify') {
+              options { skipDefaultCheckout() }
+              steps {
+                deleteDir()
+                unstash 'source'
+                unstash 'cache'
+                dir("${BASE_DIR}") {
+                  prepareRelease() {
+                    script {
+                      sh(label: 'Lerna version dry-run', script: 'lerna version --no-push --yes', returnStdout: true)
+                      def releaseVersions = sh(label: 'Gather versions from last commit', script: 'git log -1 --format="%b"', returnStdout: true)
+                      log(level: 'INFO', text: "Versions: ${releaseVersions}")
+                      emailext subject: "[${env.REPO}] Release ready to be pushed", to: "${NOTIFY_TO}",
+                               body: "Please go to ${env.BUILD_URL}input to approve or reject within 12 hours.\n Changes: ${releaseVersions}"
+                      input(message: 'Should we release a new version?', ok: 'Yes, we should.',
+                            parameters: [text(description: 'Look at the versions to be released. They cannot be edited here',
+                                              defaultValue: "${releaseVersions}", name: 'versions')])
+                    }
+                  }
+                }
               }
+            }
+            stage('Release CI') {
+              options { skipDefaultCheckout() }
               steps {
                 deleteDir()
-                dir("${OPBEANS_REPO}"){
-                  git credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba',
-                      url: "git@github.com:elastic/${OPBEANS_REPO}.git"
-                  sh script: ".ci/bump-version.sh '${env.BRANCH_NAME}'", label: 'Bump version'
-                  // The opbeans pipeline will trigger a release for the master branch
-                  gitPush()
-                  // The opbeans pipeline will trigger a release for the release tag
-                  gitCreateTag(tag: "${env.BRANCH_NAME}")
+                unstash 'source'
+                dir("${BASE_DIR}") {
+                  prepareRelease() {
+                    sh 'npm run release-ci'
+                  }
                 }
               }
+              post {
+                always {
+                  script {
+                    currentBuild.description = "${currentBuild.description?.trim() ? currentBuild.description : ''} released"
+                  }
+                }
+              }
+            }
+          }
+        }
+        stage('Opbeans') {
```

Comment:
This stage has not changed at all, but added at the very end of the pipeline as it does run when there are tag creations for the package `elastic/apm-rum`.